### DISCUSSION TITLE
Static HTML Output

### DISCUSSION THREAD

**Discussion by sirinath** [typst/typst#197] [Ideas]
Ability to output static HTML so the Typst format can be used as an alternative to MD, ADOC, etc. as a blogging platform / static site generator.

Maybe perhaps additional integration with GoHugo, GetZola, etc.

**Comment by alerque** [comment]
Duplicate of #114.

**Comment by mathcae** [comment]
We can use [Typst pandoc](https://github.com/lvignoli/typst-pandoc) to translate typst file to html
